clients/fides-js/src/components/fides.css

```
@@ -318,15 +318,26 @@ div.fides-modal-content {
 }
 
 /*Fides Embed*/
+div#fides-embed-container:focus-visible,
+div#fides-embed-container div#fides-modal:focus-visible {
+  outline: none;
+}
+
+div#fides-embed-container div#fides-modal .fides-modal-footer {
+  position: inherit;
+}
```

I'd add a comment for this one
```suggestion
/* Disable "sticky" footer when embedded */
div#fides-embed-container div#fides-modal .fides-modal-footer {
  position: inherit;
}
```